lib/matplotlib/backends/qt_editor/figureoptions.py

```diff
@@ -91,8 +96,9 @@ def figure_edit(axes, parent=None):
                          ('Edgecolor', ec),
                          ]
             curves.append([curvedata, label, ""])
-    # make sure that there is at least one displayed curve
-    has_curve = bool(curves)
+
+        # make sure that there is at least one displayed curve
+        has_curve = bool(curves)
```

Comment:
Because with out this and with no lines you get

```python
 Traceback (most recent call last):
  File "/home/tcaswell/other_source/matplotlib/lib/matplotlib/backends/backend_qt5.py", line 653, in edit_parameters
    figureoptions.figure_edit(axes, self)
  File "/home/tcaswell/other_source/matplotlib/lib/matplotlib/backends/qt_editor/figureoptions.py", line 95, in figure_edit
    has_curve = bool(curves)
UnboundLocalError: local variable 'curves' referenced before assignment

If you suspect this is an IPython bug, please report it at:
    https://github.com/ipython/ipython/issues
or send an email to the mailing list at ipython-dev@scipy.org

You can print a more detailed traceback right now with "%tb", or use "%debug"
to interactively debug it.

Extra-detailed tracebacks for bug-reporting purposes can be enabled via:
    %config Application.verbose_crash=True
```